pkg/controller/podgc/gc_controller.go

```diff
@@ -213,9 +213,11 @@ func (gcc *PodGCController) gcTerminated(ctx context.Context, pods []*v1.Pod) {
 		wait.Add(1)
 		go func(pod *v1.Pod) {
 			defer wait.Done()
+			deletingPodsTotal.WithLabelValues("terminated").Inc()
```

Comment:
I am thinking, if we should emit this metric before or after the completion of delete operation. As it stands, we emit when we start deleting.

Also, I think we should add namespace to these metrics, so as if we force deleted pods from a particular namespace, a k8s admin can investigate why that happened. @msau42 what do you think?

Reply:
Waiting for a feedback from @msau42

Reply:
namespace would be good to add. I have a slight preference to add the metric after completion. Do we care about deletion errors?

Reply:
@xing-yang what do you think？